app/src/main/java/com/github/se/gomeet/ui/mainscreens/profile/Profile.kt

```diff
@@ -292,7 +292,7 @@ fun Profile(
                     }
 
                 Spacer(modifier = Modifier.height(screenHeight / 40))
-                ProfileEventsList("Joined Events", rememberLazyListState(), myEventList, nav)
+                ProfileEventsList("Joined My Events", rememberLazyListState(), myEventList, nav)
```

Comment:
It's not the text that should change here, it's the logic, the event list needs to be the list of joined events by the user (so it's better to change the title back to what it was and change the logic). Also, there is the same problem in OthersProfile (there is a lot of copy-pasted code between these two files, maybe they should have been merged into one).